src/fixtures/PageContexts.ts

```diff
@@ -76,19 +79,59 @@ export const test = base.extend<FixtureTypes>({
ricing":{"id":10342,"mode":"gmv","name":"beyond","label":"beyond","tags":[{"id":3,"name":"REGULAR"}],"basePrice":6500,"chargingInterval":1,"monthlySurchargeIgnored":false,"includedAmount":15000000,"plannedAmount":2000000,"discountAmount":6825,"discountExpirationDate":"2025-06-17T00:00:00+02:00","pricingPeriods":[{"id":10819,"includedAmount":15000000,"plannedAmount":2000000,"discountAmount":81900,"relativeDiscountAmount":100,"periodContractValue":0,"plannedAmountFixed":true,"planBasePrice":81900,"planPrice":0,"planMonthlyPrice":0,"periodTotalPrice":0,"periodMonthlyPrice":0,"bookingPrice":0,"startDate":"2024-06-18 00:00:00","endDate":"2025-06-17 23:59:59","paasDefinition":null,"consultingServiceDefinition":null,"paasPrice":0,"consultingServicePrice":0}],"calculatedPrice":0},"charging":{"id":10342,"lastBookingDate":"2024-06-18T00:00:00+02:00","nextBookingDate":"2024-07-18T00:00:00+02:00"},"minimumDuration":12,"isUpsell":false,"isPaas":false,"hubspotDealId":null,"startDate":"2024-06-18 00:00:00","totalPlanDiscountAmount":81900,"totalPlanValue":0,"averageAnnualPlanValue":0,"monthlyPlanSurchargeAmount":0,"salesChannel":"sbp_frontend"},"hasOpenCommercialContract":false,"latestContractCancellation":null,"isCommercial":false,"domain_idn":"pw-ats-test.swstage.store","commercialEditionContract":{"id":10366,"creationDate":"2024-06-18T15:01:50+02:00","expirationDate":"2025-06-17T23:59:59+02:00","status":{"id":2,"name":"booked"},"shop":{"id":225960,"domain":"pw-ats-test.swstage.store"},"charging":{"id":10342,"lastBookingDate":"2024-06-18T00:00:00+02:00","nextBookingDate":"2024-07-18T00:00:00+02:00"},"minimumDuration":12,"isUpsell":false,"isPaas":false,"hubspotDealId":null,"startDate":"2024-06-18 00:00:00","totalPlanDiscountAmount":81900,"totalPlanValue":0,"averageAnnualPlanValue":0,"monthlyPlanSurchargeAmount":0,"salesChannel":"sbp_frontend","bookedCommercialEdition":{"id":3,"name":"beyond","label":"beyond","tags":[{"id":3,"name":"REGULAR"}],"basePrice":6500,"includedGmv":15000000,"includedOrderAmount":15000,"sortPriority":3,"isRegular":true,"isUpgrade":false,"isMigration":false,"isClient":false,"isReseller":false,"displayName":"Shopware Beyond"},"details":{"id":10342,"mode":"gmv","name":"beyond","label":"beyond","tags":[{"id":3,"name":"REGULAR"}],"basePrice":6500,"chargingInterval":1,"monthlySurchargeIgnored":false,"includedAmount":15000000,"plannedAmount":2000000,"discountAmount":6825,"discountExpirationDate":"2025-06-17T00:00:00+02:00","pricingPeriods":[{"id":10819,"includedAmount":15000000,"plannedAmount":2000000,"discountAmount":81900,"relativeDiscountAmount":100,"periodContractValue":0,"plannedAmountFixed":true,"planBasePrice":81900,"planPrice":0,"planMonthlyPrice":0,"periodTotalPrice":0,"periodMonthlyPrice":0,"bookingPrice":0,"startDate":"2024-06-18 00:00:00","endDate":"2025-06-17 23:59:59","paasDefinition":null,"consultingServiceDefinition":null,"paasPrice":0,"consultingServicePrice":0}],"calculatedPrice":0},"chargingInterval":1,"calculatedPrice":0}}',
```

Comment:
I'm a bit concerned about exposing internal software APIs in a public package. For example, is the `companyId` the real id from the account in the SBP?

This shouldn't be part of the NPM package that can be used as public dev tooling. What do you think?